jobs/nessus-agent/templates/config/crontab

```diff
@@ -0,0 +1,3 @@
+# m h dom mon dow user  command
+# Sleep a random amount of time from 0-5 minutes to avoid all agents relinking at the same time
+30 * * * * root /bin/sleep $(( ( RANDOM % 300 )  + 1 ))  && /var/vcap/jobs/nessus-agent/bin/link-agent.sh
```

Comment:
Yes, I thought it was safer to just run this regularly since it's a noop if the agent is already linked as the link-agent script checks the status before attempting a relink.